logisland-components/logisland-services/logisland-service-elasticsearch/logisland-service-elasticsearch_7_x-client/src/main/java/com/hurence/logisland/service/elasticsearch/ElasticsearchRecordConverter.java

```diff
@@ -149,6 +149,9 @@ static String convertToString(Record record, String geolocationFieldLabel) {
                     }
                 } catch (Throwable ex) {
                     logger.error("unable to process field '{}' in record : {}, {}", fieldName, record, ex.toString());
+                    if ( logger.isDebugEnabled() ) {
+                        ex.printStackTrace();
+                    }
                 }
```

Comment:
Use log method to log the stackTrace, and I do not see why in line 151 we use ex.toString instead of just passing the error itself.

Reply:
The log error can only called with (log, throwable) in this case.